frontend/src/features/frame/components/topbar/Search.tsx

```diff
@@ -55,7 +55,13 @@ const Search = () => {
     searchParams.delete('search');
     searchParams.append('search', search);
     searchParams.delete('offset');
-    history.push('/recommendations?' + searchParams.toString());
+
+    // Parse the searchParams to see if it matches a youtube video
+    // And if so stripes it UID and automatically push to the specific Tournesol page
+    const videoId = extractVideoId(search);
```

Comment:
One issue when using `extractVideoId()` on user input is that any 11-letters word will be considered as a YouTube video id. For example searching for "algorithmes" will redirect to `/entities/yt:algorithms` which is unexpected :worried:.

Maybe we should add an option to `extractVideoId()` to require the `yt:` prefix in this case. As video ids (without prefix `yt:`) are already present in the search index, the video would still be present in the search results, so I think that would be an acceptable trade-off. 

Note that the implementation of `extractVideoId()` has been slightly modified in #1847 and should be easier to maintain. Make sure to rebase on the latest "main" if you want to modify it. And rebasing would also fix the flaky test that caused the CI pipeline to fail (fixed in #1840)